src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -347,13 +390,14 @@ export const ProjectAccessTable: VFC = () => {
                                     </>
                                 }
                             />
-                            <Button
-                                variant="contained"
-                                color="primary"
+                            <ResponsiveButton
                                 onClick={() => setAssignOpen(true)}
+                                maxWidth="700px"
+                                Icon={Add}
+                                permission={UPDATE_PROJECT}
```

Comment:
This permission will need projectId and environmentId to work properly

Reply:
Good catch. Added